CondCore/Utilities/plugins/Module_2XML.cc

```diff
@@ -31,7 +31,6 @@ PAYLOAD_2XML_MODULE(pluginUtilities_payload2xml) {
   PAYLOAD_2XML_CLASS(CTPPSPixelAnalysisMask);
   PAYLOAD_2XML_CLASS(CTPPSPixelDAQMapping);
   PAYLOAD_2XML_CLASS(CTPPSPixelGainCalibrations);
-  PAYLOAD_2XML_CLASS(PPSAlignmentConfig)
```

Comment:
@MatiXOfficial 
As Far as I understand, you have kept the old condformat (PPSAlignmentConfig). In this case, better also keep the functionalities to dump the corresponding payloads in the database. This requires the PAYLOAD_2XML_CLASS macro entry.

Reply:
Ok, I restored the macro and the test.